app/views/website_members/index.html.haml

```diff
@@ -18,9 +19,13 @@
             %th.text-right Actions
         %tbody.websites
           - @website_members.each do |website_member|
-            %tr
-              %td= website_member.user.name
-              %td= website_member.user.email
-              %td= website_member.role
-              %td.text-right
-                = link_to ' Delete', "/website_members/#{website_member.id}", :method => :delete, data: {confirm: "Are you sure?"}, :class => "delete_website btn btn-danger pull-right fa fa-close"
\ No newline at end of file
+            - unless @current_website_member == website_member
+              = simple_form_for website_member, wrapper: :vertical_form do |f|
+                %tr
+                  %input{:name => "id", :type => "hidden", :value => "#{website_member.id}"}/
+                  %td= website_member.user.name
+                  %td= website_member.user.email
+                  %td
+                    = f.input :role, label: false, collection: [['Owner', 1], ['User', 2]], selected: website_member.role.value, input_html: {:style => "max-width: 90px;"}
```

Comment:
for this i think you should do something like this line of code:
```
= select_tag 'role', options_for_select(options, selected: membership.role), id: nil, class: 'ajax-spinner', style: 'width: 100%;', data: { remote: true, url: change_role_website_member_path(website_member), method: 'put', params: '_method=put', type: 'json' }
```

It takes the need for any ajax/js code that you wrote and it works out of the box